src/styles/modal.css

```diff
@@ -0,0 +1,74 @@
+/* modal */
+
+.ui.modal {
+  background-color: var(--shade-white);
+  border-radius: 0;
+  box-shadow: none;
+}
+
+.ui.modal > .icon:first-child + * {
+  border-radius: 0;
+}
+
+/* close icon */
+
+.ui.modal > .close {
+  color: var(--shade-white);
+  font-size: 1.5rem;
+  height: 1.5rem;
+  padding: 0;
+  right: var(--spacing-2);
+  top: var(--spacing-3);
+  width: 1.5rem;
+}
+
+@media only screen and (max-width: 991px) {
+  .ui.modal > .close {
+    color: var(--shade-white);
+    font-size: 1.5rem;
+    height: 1.5rem;
+    padding: 0;
+    right: var(--spacing-2);
+    top: var(--spacing-3);
+    width: 1.5rem;
+  }
+}
+
+/* title */
+
+.ui.modal > .header {
+  background: var(--primary-pms-539-900);
+  border-bottom: none;
+  color: var(--shade-white);
+  padding: var(--spacing-2) var(--spacing-3);
+}
+
+.ui.modal > .header:not(.ui) {
+  font-family: var(--gotham);
+  font-size: var(--h2-font-size);
```

Comment:
I think we need `var(--h6-font-size)` and `var(--h6-line-height)`, which is our previously named `subheading` styles.

Reply:
I'm not sure - in the design system it says `h2` 🤔
<img width="400" alt="Screenshot 2023-06-09 at 2 51 01 PM" src="https://github.com/abby-moffitt/GA-Semantic/assets/64768811/d0859d38-e52d-4fd7-8988-fde04895d9af">

Reply:
Ok, I was going off what we had in our component library, but I do see the H2 styles in the Figma files, so we can keep it as is.